.eslintrc.js

```diff
@@ -2,27 +2,21 @@ module.exports = {
   env: {
     browser: true,
     es6: true,
-    node: true,
-    mocha: true
+    node: true
   },
   extends: [
     "eslint:recommended",
     "standard"
   ],
-  globals: {
-    Atomics: "readonly",
-    SharedArrayBuffer: "readonly"
-  },
   parserOptions: {
     ecmaVersion: 2015,
     sourceType: "module"
   },
-  // parser: '@typescript-eslint/parser',
```

Comment:
Please leave, sometimes I turn this on for stricter linting of TS stuff when developing, sometimes it gets in the way.